app/models/user.rb

```diff
@@ -9,4 +9,13 @@ class User < ApplicationRecord
   def password_required?
     !persisted? || password.present? || password_confirmation.present?
   end
+
+  def member_in_sections
```

Comment:
Thank goodness - it irks me. Yes, the language is clunky and `member_of_sections` might sound a little more readable. Good point, I'll update.